tracer/src/Datadog.Trace/ClrProfiler/CallTarget/Handlers/Continuations/ValueTaskContinuationGenerator.cs

```diff
@@ -3,12 +3,12 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#if NETCOREAPP3_1 || NET5_0 || NETCOREAPP3_1_OR_GREATER
```

Comment:
Checking all 3 is redundant.
```suggestion
#if NETCOREAPP3_1_OR_GREATER
```

Reply:
I kept those in to match other places, wondered if there was still a reason to do so, and since it didn't seem harmful, I matched it.